rollup.config.js

```diff
@@ -14,21 +14,8 @@ const compiledLicense = _.template(fs.readFileSync('./build/license-header.txt',
 const bannerData = _.pick(pkg, ['version', 'copyright']);
 const banner = compiledLicense(Object.assign({includesVtt: true}, bannerData));
 
-const watch = {
-  clearScreen: false
-};
-
-const onwarn = (warning) => {
-  if (warning.code === 'UNUSED_EXTERNAL_IMPORT' ||
-      warning.code === 'UNRESOLVED_IMPORT' ||
-      (warning.code === 'UNKNOWN_OPTION' &&
-       warning.message.indexOf('progress') !== -1)) {
```

Comment:
looks like the "progress" warning is still getting printed out, https://travis-ci.org/videojs/video.js/builds/444848160#L591-L618, since imports have been fixed, we can remove that piece but keep the unknown progress option piece.

Reply:
ok